zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/processing/PartitionPauseTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.processing;
+
+import static io.camunda.zeebe.test.util.TestUtil.waitUntil;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.api.command.ClientStatusException;
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.it.clustering.ClusteringRule;
+import io.camunda.zeebe.it.util.GrpcClientRule;
+import io.camunda.zeebe.it.util.RecordingJobHandler;
+import io.camunda.zeebe.protocol.Protocol;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.test.util.BrokerClassRuleHelper;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+@ZeebeIntegration
```

Comment:
:x: This test is not using Test containers. When using `ClusteringRule`, you don't have to use `@ZeebeIntegration`. It is required when we are using `TestCluster` or `TestStandaloneBroker` to set up the Zeebe cluster. 

`ClusteringRule` is the old way. For new tests, it is preferable to use `ZeebeIntegration`. See `BrokerAdminServiceTest` as an example.

Reply:
I had confused using `ClusteringRule` and the `ZeebeIntegration` annotation.
Thanks for the information on the differences.